test/e2e/opm_test.go

```diff
@@ -288,6 +288,20 @@ var _ = Describe("opm", func() {
 				Expect(err).NotTo(HaveOccurred())
 			}
 
+			By("validating bundles")
```

Comment:
Can we potentially start separating these different things into separate tests? This is a test specifically about building bundles with `opm alpha bundle build` from a nested package manifest directory using inference for the package level parameters on each bundle.

Reply:
Sure. I can move this to its own test, but it does seem like we would want to run validate in both of the existing tests too.

Reply:
That's probably fair. I still think these tests need to be slightly more targeted in general than just "Do everything and then throw an error when something goes wrong", otherwise we will quickly get into a place where any change will require updating one of these two tests -- I think it makes sense for all of our tests to have a sizable surface area, but I'm worried about them having too many different meanings.

Reply:
I agree, though I didn't want to diverge too much from the precedent set here in a bug fix.